components/Card/Grid/NotificationGridTab.vue

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<template>
+    <ResponsiveCenteredViewTemplate :fixed="true">
+        <template #content>
+            <BaseForm>
+                <div class="notifications-list">
+                    <List v-if="notifications.length">
+                        <NotificationListElement
+                            v-for="notification in notifications"
+                            :key="notification.id"
+                            :notification="notification" />
+                        <NotificationListFooter>
+                            <BaseButton
+                                :title="buttonTitle"
+                                @click.native="onLoadMoreNotifications" />
+                        </NotificationListFooter>
+                    </List>
+                    <NotificationPlaceholder
+                        v-else
+                        :layout-orientation="horizontalOrientation" />
+                </div>
+            </BaseForm>
+        </template>
+    </ResponsiveCenteredViewTemplate>
+</template>
+
+<script>
+import { mapState, mapActions } from 'vuex';
+import { LayoutOrientation } from '~/defaults/layout';
+import defaultState from '~/store/notifications/state';
```

Comment:
isn't it better to use the default variable LIMIT and use this in store and here.